rubocop-airbnb/config/rubocop-layout.yml

```diff
@@ -227,7 +227,7 @@ Layout/FirstMethodParameterLineBreak:
   Enabled: false
 
 # Supports --auto-correct
-Layout/IndentFirstArgument:
+Layout/FirstArgumentIndentation:
```

Comment:
Looks like we lost the custom renames of these rules. oth, i don't know why we wanted to customize this by renaming rules anyway. Seems like a low ROI and high maintenance cost to me. Would recommend we ditch our customizations, but probably want to check w/ John. Also, they changed the format so the old kindsys renames would be inconsistent anyway.

afaict, there are 3 renames in the kindsys customizations (note that the source rule names have changed):
Layout/FirstParameterIndentation -> Layout/IndentFirstArgument
Layout/IndentArray -> Layout/IndentFirstArrayElement
Layout/IndentHash -> Layout/IndentFirstHashElement